README.md

```diff
@@ -32,6 +32,7 @@ Here are the different groups of integrations:
 - the **Front-End Integrations**: to integrate a smooth search experience in the front-end of the projects.
 - the **Web Framework Integrations**: to connect the web frameworks with MeiliSearch.
 - the **DevOps Tools**: to make the deployement of MeiliSearch easier.
+- the **Platform Plugins**: to sync your data with MeilISearch and or add a search experience on different platforms.
```

Comment:
```suggestion
- the **Platform Plugins**: to sync your data with MeiliSearch and/or add a search experience on different platforms.
```

Reply:
First I did `and/or` but Grammarly screamed at me as apparently, it is `very frowned upon`. But MeiliSearch is definitely a typo

Reply:
I had to read twice the sentence to understand because of the `and or`, and my brain conclusion was first "there is an extra word" and then, I understand. I'm not an expert in English, so maybe I'm wrong.